src/main/java/frc/robot/RobotContainer.java

```diff
@@ -283,6 +298,11 @@ private void configureButtonBindings() {
           }, swerveDrive));
       
       balanceChargingStationButton.whileTrue(levelChargingStationCommand);
+      balanceChargingStationButton.whileTrue(new InstantCommand(
```

Comment:
Move this `InstantCommand` to the LevelChargingStation `addCommands` function instead of attaching it to the button here